standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/Warehouse.java

```diff
@@ -133,14 +133,8 @@ public FileSystem getFs(Path f) throws MetaException {
 
 
   /**
-   * Hadoop File System reverse lookups paths with raw ip addresses The File
-   * System URI always contains the canonical DNS name of the Namenode.
-   * Subsequently, operations on paths with raw ip addresses cause an exception
-   * since they don't match the file system URI.
```

Comment:
I realy don't understand what's meaning of this comment.
https://github.com/apache/hive/blob/master/itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestWarehouseDnsPath.java The tests show that `getDnsPath` has nothing to do with `Hadoop File System reverse lookups`.

IMO, the hdfs filesystem should be same as object stores, s3a/s3n, we can also optimizie hdfs fs like HIVE-24838.  https://github.com/apache/hive/pull/2085#issue-833982804 also said `getDnsPath` : `The only useful thing this function does is transforming a relative path to an absolute path.`

I hope someone can help me to understand this comment about the `Hadoop File System reverse lookups paths`.
@ayushtkn Do you have any thought?